utils/_booking.py

```diff
@@ -30,10 +30,32 @@ def __hash__(self):
 
 
 class Dataset:
-    def __init__(self, name, ntuples):
+    def __init__(self, name, ntuples, quantities_per_vars):
```

Comment:
Did you test if the updated code still works on the ntuples created with Artus? This change looks to me like it would break backwards compatibility as there is no default value given and the call in `booking.py` is not updated. 
```suggestion
    def __init__(self, name, ntuples, quantities_per_vars=None):
```